dts/arm/nxp/nxp_rt1064.dtsi

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019, Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_rt.dtsi>
+
+&flexram0 {
+	ocram1: ocram@20280000 {
```

Comment:
RT1060 needs the same change. The difference between RT1060 and RT1064 is that the latter adds internal flash.

@galak, should we handle this with separate `nxp_rt1060.dtsi` and `nxp_rt1064.dtsi` files, or combine into `nxp_rt106x.dtsi`? I'm leaning toward combined.

Reply:
What the difference between RT1060 & RT1064.  I'm probably same combinded, and at some point we can introduce an SoC set if needed.